easybuild/easyblocks/c/cuda.py

```diff
@@ -116,3 +147,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
```

Comment:
this is not needed anymore, `$CUDA_HOME` will now be defined as it should thanks to the bug fix in https://github.com/hpcugent/easybuild-framework/pull/1519

so, drop this whole method